pom.xml

```diff
@@ -622,6 +623,7 @@
                     <configuration>
                         <autoVersionSubmodules>true</autoVersionSubmodules>
                     </configuration>
+                    <version>2.5.1</version>
```

Comment:
Fixed, I was experimenting with features and wanted to make sure I knew what version I was screwing with. Accidental settings leaving in